mne_bids/utils.py

```diff
@@ -164,9 +165,12 @@ def make_bids_folders(subject, session=None, kind=None, root=None,
     make_dir : bool
         Whether to actually create the folders specified. If False, only a
         path will be generated but no folders will be created.
-    overwrite : bool
-        If `make_dir` is True and one or all folders already exist,
-        this will overwrite them with empty folders.
+    write_mode : str, one of ('append', 'overwrite', 'error')
+        How to handle overwriting previously generated data.
+        If write_mode == 'append' or 'error' the previously created folder
+        will be left intact.
```

Comment:
I hope I'm not going around in circles. But `write_mode = 'append'` should not be an option for `make_bids_folders`, no?

Reply:
oh yeah, I can do what I did for the `_mkdir_p` function and remove the `write_mode` up one more level. Ie. what I did here:

https://github.com/mne-tools/mne-bids/pull/99/files/11465ad16a234f528cc9137e6373824a5008bcb2#diff-57fc2a5d75c57bb143f494b0637ca740R204